pkg/kubectl/apply/element.go

```diff
@@ -325,6 +326,18 @@ func IsAdd(e Element) bool {
 	return e.HasLocal() && !e.HasRemote()
 }
 
+// IsConflict returns true if the object represented by e should either have the same field specified
```

Comment:
We can simplify the comment by saying:
"returns true if the field represented by e has conflict between remote and recorded value".

Reply:
nice!